### PR TITLE
Feature/university list screen

### PR DESCRIPTION
**Pull request by Uticodes**
# PR Description:

## Title:

Implement Universities List Screen

## Description:

- Design and Implement the list items
- Design and Implement Search Input

## Screenshot

https://github.com/Uticodes/UniversityList/assets/43546652/f2042417-a0c4-4bc5-86b8-abe3509fb7f7

### CODE REVIEW

File: app/src/main/java/com/example/universitieslist/view/components/UniversityItem.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+@Composable
+fun UniversityItem(schoolName: String, website: String, country: String) {
+    Card(
+        modifier = Modifier.padding(top = dimens12),
+        elevation = CardDefaults.cardElevation(dimens6),
+        colors = CardDefaults.cardColors(containerColor = Purple40)
+    ) {
+        Column(modifier = Modifier.padding(dimens8)) {
+            TextItem(
+                hintText = "Name",
```
Comment (princessdharmy) [inline app/src/main/java/com/example/universitieslist/view/components/UniversityItem.kt:32]: This string needs to be extracted as a resource. Same with others.

File: app/src/main/java/com/example/universitieslist/view/components/UniversityItem.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+@Composable
+fun TextItem(
+    hintText: String,
+    title: String,
+    textFont: TextUnit = FontSize.fontSize12
+) {
+    Row(modifier = Modifier.fillMaxWidth()) {
+        Text(
+            text = "$hintText:",
```
Comment (princessdharmy) [inline app/src/main/java/com/example/universitieslist/view/components/UniversityItem.kt:56]: Why not add the semicolon with the hint text in string resource value?

File: app/src/main/java/com/example/universitieslist/view/components/UniversityItem.kt (hunk 1 above)
Comment (Uticodes) [reply app/src/main/java/com/example/universitieslist/view/components/UniversityItem.kt]: Done

File: app/src/main/java/com/example/universitieslist/view/components/UniversityItem.kt (hunk 2 above)
Comment (Uticodes) [reply app/src/main/java/com/example/universitieslist/view/components/UniversityItem.kt]: Done

**Review by princessdharmy — APPROVED** [review]

**Pull request merged**